addons/crm/static/src/js/forecast/forecast_kanban_column_quick_create.js

```diff
@@ -0,0 +1,56 @@
+/** @odoo-module alias=crm.ForecastKanbanColumnQuickCreate **/
+import Widget from 'web.Widget';
+import {_t} from 'web.core';
+ 
+/**
+ * File intended for `web` (with forecast_kanban.xml) (can be used to add next "temporal" group)
+ * 
+ * This file defines the ColumnQuickCreate widget for ForecastKanban. It allows to
+ * create kanban columns directly from the Kanban view with a dedicated
+ * column creation logic coming from the view
+ */
+
+var ForecastColumnQuickCreate = Widget.extend({
+    template: 'KanbanView.ForecastColumnQuickCreate',
+    events: {
+        'click .o_quick_create_folded': '_onAddClicked',
+    },
+
+    /**
+     * @override
+     * @param {Object} [options]
+     * @param {string} options.granularity displayed granularity
+     */
+    init: function (parent, options) {
+        this._super.apply(this, arguments);
+        this.granularity = `${_t('Add next')} ${options.granularity}`;
```

Comment:
Could be great to include the variable name in translated sentence like `_t('Add next ${granularity}')` ? That way it is exported and people may move variable name if it suits better in their language.